src/EA.Weee.RequestHandlers/Admin/DirectRegistrants/ReturnSmallProducerSubmissionHandler.cs

```diff
@@ -52,7 +52,7 @@ private static void ValidateRequest(ReturnSmallProducerSubmission request)
         private void EnsureAuthorisation()
         {
             authorization.EnsureCanAccessInternalArea();
-            authorization.EnsureUserInRole(Roles.InternalUser);
+            authorization.EnsureUserInRole(Roles.InternalAdmin);
```

Comment:
@will-hope98  Suspect ReturnSmallProducerSubmissionHandlerIntegrationTestBase will need to be updated .WithInternalUserAccess(true). Also suspect the unit test will need doing if not updated below somewhere...